.changie.yaml

```diff
@@ -0,0 +1,50 @@
+changesDir: .changes
+unreleasedDir: unreleased
+headerPath: header.tpl.md
+versionHeaderPath: ""
+changelogPath: CHANGELOG.md
+versionExt: md
+versionFormat: '## dbt-core {{.Version}} - {{.Time.Format "January 02, 2006"}}'
+kindFormat: '### {{.Kind}}'
+changeFormat: '- {{.Body}} ([#{{.Custom.Issue}}](https://github.com/dbt-labs/dbt-core/issues/{{.Custom.Issue}}), [#{{.Custom.PR}}](https://github.com/dbt-labs/dbt-core/pull/{{.Custom.PR}}))'
+kinds:
+- label: Fixes
+- label: Features
+- label: Under the Hood
+- label: Breaking Changes
+- label: Docs
+- label: Dependencies
+custom:
+- key: Author
+  label: GitHub Name
+  type: string
+  minLength: 3
+- key: Issue
+  label: GitHub Issue Number
+  type: int
+  minLength: 4
+- key: PR
+  label: GitHub Pull Request Number
+  type: int
+  minLength: 4
+footerFormat: |
+  Contributors:
+  {{- $contributorDict := dict }}
+  {{- $core_team := list "emmyoop" "nathaniel-may" "gshank" "leahwicz" "ChenyuLInx" "stu-k" "iknox-fa" "VersusFacit" "McKnight-42" "jtcohen6" }}
```

Comment:
This can be a good first issue for new hires to just add themselves to this list

Reply:
Is there a single running list of things like this somewhere?

I don't like that this is hardcoded but I'm not seeing a way around it at the moment.

Reply:
The [Github API](https://docs.github.com/en/rest/reference/teams#list-team-members) looks like it has a way to do it via the returned "login" field, but it would involve mapping over the returned list. I agree it would be good for a subsequent improvement.